src/shared/cloudformation/templateSymbolResolver.ts

```diff
@@ -4,13 +4,16 @@
  */
 
 import { safeLoad } from 'js-yaml'
-import * as _ from 'lodash'
 import * as vscode from 'vscode'
 import { CloudFormation } from './cloudformation'
 
+/**
+ * SAM template Lambda resource or API Gateway resource.
+ */
 export interface TemplateFunctionResource {
     name: string
     range: vscode.Range
+    kind: 'function' | 'api'
```

Comment:
I considered it and intentionally did not do so, because the use of this is local to this module and it adds more indirection/ceremony to define yet another symbol.

Reply:
Sounds reasonable.